tests/attr/layer/test_layer_lrp.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+import torch
+import torch.nn as nn
+
+from captum.attr import LayerLRP
+from captum.attr._utils.lrp_rules import Alpha1_Beta0_Rule, EpsilonRule, GammaRule
+
+from ...helpers.basic import BaseTest, assertTensorAlmostEqual
+from ...helpers.basic_models import BasicModel_ConvNet_One_Conv, SimpleLRPModel
+
+
+def _get_basic_config():
+    input = torch.arange(16).view(1, 1, 4, 4).float()
+    return BasicModel_ConvNet_One_Conv(), input
+
+
+def _get_rule_config():
```

Comment:
nit: is this being used anywhere ?

Reply:
True, it is not used anymore. I removed it now.